roles/ceph-mon/tasks/docker/fetch_configs.yml

```diff
@@ -20,7 +19,7 @@
 
 - name: try to fetch ceph config and keys
   copy:
-    src: "{{ fetch_directory }}/docker_mon_files/{{ item.0 }}"
+    src: "{{ playbook_dir }}/{{ fetch_directory }}/{{ fsid }}/{{ item.0 }}"
```

Comment:
I'm afraid fsid will be empty since ceph-common role is never called.
So where do you get it from?